code/datums/uplink/hardsuits and modules.dm

```diff
@@ -0,0 +1,166 @@
+
+/datum/uplink_item/item/hardsuit_modules/suit/elyra
+	name = "Elyran Battlesuit"
+	telecrystal_cost = 8
+	bluecrystal_cost = 8
+	path = /obj/item/rig/elyran
+	desc = "An advanced Elyran hardsuit specialized in scorched earth tactics. Only wearable by humans."
+
+/datum/uplink_item/item/hardsuit_modules/suit/dominia
+	name = "Dominian Jinxiang Hardsuit"
+	telecrystal_cost = 8
+	bluecrystal_cost = 8
+	path = /obj/item/rig/jinxiang
+	desc = "An off-shoot of the core Bunker Suit design, utilized by the Imperial Dominian military and painted accordingly. This is a powerful suit specializing in melee confrontations. Only wearable by humans and unathi."
+
+/datum/uplink_item/item/hardsuit_modules/suit/tcaf
+	name = "TCAF Legionnaire Hardsuit"
+	telecrystal_cost = 8
+	bluecrystal_cost = 8
+	path = /obj/item/rig/combat/legionnaire
+	desc = "An armored combat hardsuit in the blue colors of the Tau Ceti Armed Forces. The red shoulder pad dignifying the individual as a member of rank. \
+	Its golden visor reflecting the shining liberty the TCAF stands for. Only wearable by humans, Zeng-hu IPCs and Bishop IPCs."
+
+/datum/uplink_item/item/hardsuit_modules/suit/xanu
+	name = "Xanan dNAXS-52 Hardsuit"
+	telecrystal_cost = 8
+	bluecrystal_cost = 8
+	path = /obj/item/rig/xanu
+	desc = "The dNAXS-52 combat hardsuit is designed for the All-Xanu Spacefleet's interstellar infantry. It is specially designed for boarding operations, close quarters combat, and demolitions. Only wearable by humans and non-industrial IPCs."
+
+/datum/uplink_item/item/hardsuit_modules/suit/crimson
+	name = "Crimson Hardsuit"
+	telecrystal_cost = 8
+	bluecrystal_cost = 8
+	path = /obj/item/rig/merc/trulyempty
+	desc = "A blood-red hardsuit featuring some fairly illegal technology. Made by Hammertail, and popular on the black market. Only wearable by humans, unathi, skrell, tajara and non-industrial IPCs."
+
+/datum/uplink_item/item/hardsuit_modules/suit/einstein
+	name = "Einstein Engines Paragon Hardsuit"
+	telecrystal_cost = 8
+	bluecrystal_cost = 8
+	path = /obj/item/rig/merc/einstein/empty
+	desc = "A back mounted control mechanism of an Einstein Engines hardsuit. This model is issued to the leaders of security teams within the corporation. Wearable only by humans, industrial, zeng-hu and bishop IPCs."
+
+/datum/uplink_item/item/hardsuit_modules/suit/qukala
+	name = "Qukala Assault Hardsuit"
+	telecrystal_cost = 8
+	bluecrystal_cost = 8
+	path = /obj/item/rig/skrell
+	desc = "A Nralakk-manufactured combat hardsuit, designed for use by elite operatives of the Qukala. Due to their expense and classified design, these suits are rarely seen outside of Qukala hands. Only wearable by skrell."
+
+/datum/uplink_item/item/hardsuit_modules/suit/breacher // These three suits, have better armor than the others, so are priced at 10 tc. The highest possible cost to still allow buying their factions kit.
+	name = "Unathi Breacher Hardsuit"
+	telecrystal_cost = 10
```

Comment:
not really sold on this, nothing the horizon has can penetrate this armour

these also come with integrated power fists